demo_scripts/AKSDeployment/README.md

```diff
@@ -176,7 +175,7 @@ Grab the Vnet ID by running the following:
 AKS_VNET_ID=$(az network vnet show -n $AKS_VNET_NAME -g $NODE_RESOURCE_GROUP -o tsv --query "id")
 ```
 Create the peering from Application Gateway to AKS by runnig the following:
-'az network vnet peering create --name AppGWtoAKSVnetPeering -g $RESOURCE_GROUP_NAME --vnet-name'
+'az network vnet peering create --name AppGWtoAKSVnetPeering -g $RESOURCE_GROUP_NAME --vnet-name $VNET_NAME --remote-vnet $AKS_VNET_ID --allow-vnet-access'
```

Comment:
Sorry I missed this on the first pass, there's a hard coded value in here. "ppGWtoAKSVnetPeering" and we should be using full parameter names for readability "-g"